grpc/src/test/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilderTest.java

```diff
@@ -244,6 +247,43 @@ void addServices() {
         }
     }
 
+    @Test
+    void setGrpcHealthCheckService() {
+        final GrpcService grpcService =
+                GrpcService.builder()
+                           .grpcHealthCheckService(new GrpcHealthCheckService(
+                                   ImmutableSet.of(),
+                                   ImmutableMap.of(),
+                                   ImmutableList.of()
+                           ))
+                           .build();
+        assertThat(grpcService.services().stream().map(it -> it.getServiceDescriptor().getName()))
+                .containsExactlyInAnyOrderElementsOf(ImmutableList.of("grpc.health.v1.Health"));
+    }
+
+    @Test
+    void enableDefaultGrpcHealthCheckService() {
+        final GrpcService grpcService =
+                GrpcService.builder()
+                           .enableGrpcHealthCheckService(true)
+                           .build();
+        assertThat(grpcService.services().stream().map(it -> it.getServiceDescriptor().getName()))
+                .containsExactlyInAnyOrderElementsOf(ImmutableList.of("grpc.health.v1.Health"));
+    }
+
+    @Test
+    void illegalStateOfGrpcHealthCheckService() {
+        assertThrows(IllegalStateException.class,
```

Comment:
nit; we prefer to use `assertj` for the richer API, better style, and more descriptive error messages
Can you use `assertThatThrownBy`?